client/src/pages/Profiles/ProfileCard/useStyles.ts

```diff
@@ -0,0 +1,39 @@
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles(() => ({
+  card: {
+    marginRight: 40,
+  },
+  cardContainer: {
+    height: 275,
+    width: 225,
+    // boxShadow: '1px 1px 5px lightgrey, -1px -1px 5px lightgrey',
+    // marginRight: 40,
+    textAlign: 'center',
+    position: 'relative',
+  },
+  profilePic: {
+    height: 80,
+    width: 80,
+    borderRadius: '50%',
+    marginTop: 0,
+  },
+  description: {
+    width: '70%',
+  },
+  innerFlexRowProfileContainer: {
+    position: 'absolute',
+    bottom: 10,
+    marginTop: '30px',
+    width: '80%',
+  },
+  location: {
+    marginRight: '60px !important',
```

Comment:
I wouldn't recommend using the `!important` flag to override CSS styles, let's remove it

Ditto throughout